test/db/asm/evm

```diff
@@ -0,0 +1,258 @@
+d "STOP" 00
+d "ADD" 01
+d "SUB" 03
+d "DIV" 04
+d "SDIV" 05
+d "MOD" 06
+d "SMOD" 07
+d "ADDMOD" 08
+d "MULMOD" 09
+d "EXP" 0a
+d "SIGNEXTEND" 0b
+d "GT" 11
+d "SLT" 12
+d "SGT" 13
+d "EQ" 14
+d "ISZERO" 15
+d "AND" 16
```

Comment:
as explained before, there is no need to duplicate the tests.
just have `ad "STOP" 00` to test the decoding and encoding of the opcode.